.gitignore

```diff
@@ -32,12 +32,6 @@
 # Dependency directories (remove the comment below to include it)
 # vendor/
 
-# RPM/DEB
```

Comment:
@sgagniere These should be added at the top of the file above the section that's managed by service bot